app/views/courses/TeacherClassesView.coffee

```diff
@@ -11,14 +11,68 @@ CourseInstances = require 'collections/CourseInstances'
 ClassroomSettingsModal = require 'views/courses/ClassroomSettingsModal'
 CourseNagSubview = require 'views/teachers/CourseNagSubview'
 InviteToClassroomModal = require 'views/courses/InviteToClassroomModal'
+Prepaids = require 'collections/Prepaids'
 User = require 'models/User'
 utils = require 'core/utils'
+
 helper = require 'lib/coursesHelper'
 
+translateWithMarkdown = (label) ->
+  marked.inlineLexer $.i18n.t(label), []
+
 module.exports = class TeacherClassesView extends RootView
   id: 'teacher-classes-view'
   template: template
   helper: helper
+  translateWithMarkdown: translateWithMarkdown
+
+  # TODO: where to track/save this data?
```

Comment:
The programming-idealist in me thinks it probably goes in the database. If not, then a helper file (I forget where in the folder structure would be most appropriate)